recipe/meta.yaml

```diff
@@ -12,18 +12,19 @@ source:
   sha256: {{ checksum }}
 
 build:
+  noarch: python
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-  build:
+  host:
     - python
-    - setuptools
+    - pip
     - setuptools_scm >=1.15.0
 
   run:
     - python
-    - importlib  # [py26]
+    - importlib
```

Comment:
This should just be dropped, right? Python 2.7+ has this IIUC.

Reply:
Yes, thanks